packages/server/src/adapters/aws-lambda/utils.ts

```diff
@@ -140,12 +139,47 @@ export function getPath(event: APIGatewayEvent) {
   });
 }
 
+export function getURLFromEvent(event: APIGatewayEvent): URL {
```

Comment:
No idea if this works for the AWS adapter "for real" -- need to source someone deploying on AWS